save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/utils/HttpUtils.kt

```diff
@@ -30,15 +31,19 @@ import okio.buffer
 import okio.use
 import kotlin.native.concurrent.AtomicLong
 
+private const val DOWNLOAD_REQUEST_TIMEOUT_MILLIS = 5 * 60 * 1000L
```

Comment:
suggestion to use more readable format:
```suggestion
private val DOWNLOAD_REQUEST_TIMEOUT_MILLIS = 5.minutes.toLong(DurationUnit.MILLISECONDS)
```